masterlist.yaml

```diff
@@ -722,11 +799,19 @@ plugins:
       - 'EBM_UnlimitedPower_Vampire.esp'
       - 'EBM_UnlimitedPower_Werewolf.esp'
   - name: 'SkyUI_SE.esp'
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/12604/']
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12604/'
+        name: 'SkyUI SE on Nexus Mods'
```

Comment:
I could have guessed the mod's name from the plugin, but it's fine to leave this in.

Reply:
I think I added a link and name for all originally, but ended up taking them out where there was only one link or name could be guessed. Missed this one.